src/utils/api.js

```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
+const newsApi = axios.create({
+  baseURL: "https://backend-news-project-drc2.onrender.com",
+});
+
+export function getArticles() {
+  return newsApi
+    .get("/api/articles")
+    .then((articles) => {
+      return articles.data.results;
+    })
+    .catch((err) => {
+      console.log(err);
+    });
```

Comment:
Extracted api function looks good - you will want to remove the catch from here and move it into the associated component when the time comes